<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Folders</class>
 <widget class="QWidget" name="Folders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="includeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Include</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftAllPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defaultPushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QListWidget" name="includeListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightAllPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="excludeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Exclude</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="searchAllFoldersCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Search all directories</string>
   </property>
  </widget>
  <widget class="QListWidget" name="excludeListWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>161</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
